ush/config.sh.RRFS_CONUS_13km

```diff
@@ -0,0 +1,244 @@
+MACHINE="jet"
```

Comment:
Should this file be included in RRFS_dev1? If so, please make sure to clean it up to remove all personal directories and point to the location of the real time run.

Reply:
I made this configuration with small grid dimension for saving the resources when we learn, test, and develop the system. This is not for real-time run but need to be there for everyone to try on their own space.

Reply:
Please remove the paths to your output directories.

Reply:
OK